api/src/app/app.module.ts

```diff
@@ -70,4 +70,6 @@ function getLogLevel():string {
 })
 export class AppModule {
   constructor(private appConfigService: AppConfigService) {}
-}
\ No newline at end of file
+}
+
+// add a comment to trigger api deployment
```

Comment:
just add this comment so the api deployment will trigger and we can check the public site, I'll remove this comment in the next pr for adding the popup